pkg/rateengine/rateengine_test.go

```diff
@@ -139,10 +140,14 @@ func (suite *RateEngineSuite) computePPMIncludingLHRates(originZip string, desti
 		SITRate:                         unit.NewDiscountRateFromPercent(50.0),
 	}
 	suite.MustSave(&tspPerformance)
+	useFallBackDate := false
```

Comment:
can we use consistent naming here?
```suggestion
	allowBookDate := false
```

Reply:
updating